flintrock/flintrock.py

```diff
@@ -334,6 +339,22 @@ def launch(
             )
         services += [spark]
 
+    # Format ec2 tags
+    if ec2_tags:
+        for ec2_tag in ec2_tags:
+            if ec2_tag.count(',') != 1:
+                raise UsageError(
```

Comment:
If you convert this into a function that gets called via a Click callback, it might be good to replace this with [`BadParameter`](http://click.pocoo.org/6/api/#click.BadParameter).